html-lists/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>HTML Lists</title>
+  </head>
+  <body>
+    <div>
+      <h1>HTML Lists</h1>
+      <h2>Ordered List Example</h2>
+        <ol>
+          <li>Learn to code.</li>
+          <li>Build projects.</li>
+          <li>Get a job.</li>
+        </ol>
+      <h2>Unordered List Example</h2>
+        <ul>
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JS</li>
+          <li>SQL</li>
+          <li>PHP</li>
+        </ul>
```

Comment:
Same thing here, this should be even with the above `h2` in indentation

Reply:
changes were made.